api/api.go

```diff
@@ -62,12 +62,39 @@ const (
 	OptTimeoutSec = "TimeoutSec"
 	// OptQuiesceID query parameter use for quiesce
 	OptQuiesceID = "QuiesceID"
+	// OptCredUUID is the UUID of the credential
+	OptCredUUID = "CredUUID"
+	// OptCredType  indicates type of credential
+	OptCredType = "CredType"
+	// OptCredEncrKey is the key used to encrypt data
+	OptCredEncrKey = "CredEncrypt"
+	// OptCredRegion indicates the region for s3
+	OptCredRegion = "CredRegion"
+	// OptCredDisableSSL indicated if SSL should be disabled
+	OptCredDisableSSL = "CredDisableSSL"
+	// OptCredEndpoint indicate the cloud endpoint
+	OptCredEndpoint = "CredEndpoint"
+	// OptCredAccKey for s3
+	OptCredAccessKey = "CredAccessKey"
+	// OptCredSecretKey for s3
+	OptCredSecretKey = "CredSecretKey"
+	// OptCredGoogleProjectID projectID for google cloud
+	OptCredGoogleProjectID = "CredProjectID"
+	// OptCredGoogleJsonKey for google cloud
+	OptCredGoogleJsonKey = "CredJsonKey"
+	// OptCredAzureAccountName is the account name for
+	// azure as the cloud provider
+	OptCredAzureAccountName = "CredAccountName"
+	// OptOptCredAzureAccountKey is the accountkey for
+	// azure as the cloud provider
+	OptCredAzureAccountKey = "CredAccountKey"
```

Comment:
Seems some of these parameters are similar (azure account key == google json key, azure account name == google project id == was access key ? )

Reply:
Better to name those parameters as they are called.. otherwise it can be confusing.